skeletons/INTEGER.h

```diff
@@ -43,6 +43,8 @@ xer_type_encoder_f INTEGER_encode_xer;
 per_type_decoder_f INTEGER_decode_uper;
 per_type_encoder_f INTEGER_encode_uper;
 
+#define INTEGER_constraint asn_generic_no_constraint
```

Comment:
I confess to being thoroughly confused now. I understood that your `+#define INTEGER_constraint asn_generic_no_constraint` was needed because the `sample.source.PKIX1` failed to work for you without it? And I showed that it was not the case as that sample code worked for me.

???